lib/tapioca/compilers/dsl/active_record_typed_store.rb

```diff
@@ -0,0 +1,171 @@
+# typed: true
+# frozen_string_literal: true
+
+require "parlour"
+require "tapioca/core_ext/class"
+
+begin
+  require "activerecord-typedstore"
+  require "active_record"
+  require "action_controller"
```

Comment:
Why do we need this?

Reply:
`action_controller` is just a leftover, I forgot to remove it.